provisioner/template.go

```diff
@@ -839,3 +840,9 @@ func scaleQuantity(quantityStr string, factor float32) (string, error) {
 	}
 	return quantity.String(), nil
 }
+
+// clusterName returns the canonical cluster name based on the cluster
+// provider.
+func clusterName(cluster api.Cluster) (string, error) {
```

Comment:
This doesn't need to return an error.

Reply:
I assume we can't use `cluster.Name()` in the templates directly?

Reply:
Dropped this function as we can use `.Cluster.Name` in templates. The test is updated to use this.